lib/generator/inputLatencyFrameThroughputGenerator.py

```diff
@@ -0,0 +1,277 @@
+                return_result[input_generator_name]['event_tags'][event_obj['event']] = convert_to_dct(search_target_fp, event_obj['fraction'])
+        elif input_sample_index == 2:
+            for event_obj in CombinationGenerator.BROWSER_VISUAL_EVENT_POINTS['forward_search']:
+                search_target_fp = crop_data_dict[event_obj['search_target']]['fp_list'][0]['output_fp']
+                return_result[input_generator_name]['event_tags'][event_obj['event']] = convert_to_dct(search_target_fp, event_obj['fraction'])
+
+        return return_result
+
+    def crop_images_based_on_samplefiles(self, input_data):
+        """
+
+        @param input_data:
+        @return:
+        """
+
+        # compare source image list
+        input_image_list = copy.deepcopy(input_data['converter_result'])
+
+        # generate image list
+        image_list = []
+        for image_fn in input_data['converter_result']:
+            image_list.append(input_data['converter_result'][image_fn])
+
+        # generate crop data for all images
+        crop_data_dict = generate_crop_data(image_list, input_data['sample_result'][1][self.__class__.__name__]['crop_data'])
+
+        # crop images
+        start_time = time.time()
+        crop_images(crop_data_dict)
+        last_end = time.time()
+        elapsed_time = last_end - start_time
+        logger.debug("Crop Image Time Elapsed: [%s]" % elapsed_time)
+
+        # merge crop data and convert data
+        for crop_target_name in crop_data_dict:
+            for crop_img_obj in crop_data_dict[crop_target_name]['fp_list']:
+                image_fn = os.path.basename(crop_img_obj['input_fp'])
+                input_image_list[image_fn][crop_target_name] = crop_img_obj['output_fp']
+
+        return input_image_list
+
+    def generate_result(self, input_data):
+        """
+
+        @param input_data:
+        @return:
+        """
+
+        self.compare_result = {}
+
+        input_image_list = self.crop_images_based_on_samplefiles(input_data)
+        self.compare_result['merged_crop_image_list'] = input_image_list
+
+        # compare images
+        compare_setting = {'default_fps': self.index_config['video-recording-fps'],
+                           'event_points': self.BROWSER_VISUAL_EVENT_POINTS,
+                           'generator_name': self.__class__.__name__,
+                           'skip_status_bar_fraction': self.SKIP_STATUS_BAR_FRACTION,
+                           'exec_timestamp_list': input_data['exec_timestamp_list'],
+                           'threshold': self.index_config.get('compare-threshold', 0.0003),
+                           'search_margin': self.index_config.get('search-margin', 10)}
+        self.compare_result['running_time_result'] = compare_with_sample_image_multi_process(
+            input_data['sample_result'],
+            input_image_list,
+            compare_setting)
+        # get frame throughput values
```

Comment:
`il_end` and `ft_start`(doesn't defined yet) should have one frame diff because we choose `il_end` has shift mechanism and ft calculation doesn't. for time consuming concern we should post-process after find `[il_start, il_end, and ft_end]` finished